internal/exec/stages/files/files.go

```diff
@@ -60,19 +60,29 @@ func (creator) Name() string {
 type stage struct {
 	util.Util
 	toRelabel []string
+	isLive    bool
 }
 
 func (stage) Name() string {
 	return name
 }
 
+func (s stage) ApplyLive(config types.Config) error {
+	s.isLive = true
+	return s.Run(config)
+}
+
 func (s stage) Run(config types.Config) error {
-	if err := s.checkRelabeling(); err != nil {
-		return fmt.Errorf("failed to check if SELinux labeling required: %v", err)
-	}
+	if !s.isLive {
+		if err := s.checkRelabeling(); err != nil {
+			return fmt.Errorf("failed to check if SELinux labeling required: %v", err)
+		}
 
-	if err := s.createPasswd(config); err != nil {
-		return fmt.Errorf("failed to create users/groups: %v", err)
+		// theoretically could support this, but the main user (CoreOS
+		// layering) does not
```

Comment:
In that case we should fail if the config has any `passwd` entries, ideally in a way that integrates nicely with `IgnoreUnsupported`.

Reply:
This required lowering `IgnoreUnsupported` into the stages, but I think it makes sense.